### DISCUSSION TITLE
Where is the ClangSharp code to generate the types?

### DISCUSSION THREAD

**Discussion by jzabroski** [Microsoft/LLVMSharp#193] [Q&A] [answered]
@tannergooding I tried finding where the magic happens.  Is that in this repo?

**Comment by tannergooding** [comment, marked as answer]
The code for ClangSharp lives in the ClangSharp repo: https://github.com/dotnet/clangsharp

For legacy reasons the rsp file used to regenerate LLVMSharp lives there too: https://github.com/dotnet/ClangSharp/blob/main/sources/ClangSharpPInvokeGenerator/Properties/GenerateLLVM.rsp
* These reasons don't really matter and I should fix, just haven't done it yet -- PRs welcome ;)
